### PR TITLE
Changes necessary to remove compilation errors 

### PR DESCRIPTION
**Pull request by aasingh**
Summary of changes 
1. Adding missing attribute IDs and changing attribute IDs to remove compilation errors, for example "Unknown ID or title "_export-import", used as an internal cross reference
Unknown ID or title "fake/../../client.adoc", used as an internal cross reference"

2. Replacing instances of Keycloak with {{book.project.name}}

### CODE REVIEW

File: topics/MigrationFromOlderVersions.adoc [hunk 1]
```diff
@@ -68,10 +68,10 @@ We've moved the themes and providers directories from `standalone/configuration/
 If you have added custom themes and providers you need to move them to the new location.
 You also need to update `keycloak-server.json` as it's changed due to this. 
 
-===== Adapter Subsystems only bring in dependencies if keycloak is on
+===== Adapter Subsystems only bring in dependencies if {{book.project.name}} is on
 
-Previously, if you had installed our saml or oidc keycloak subsystem adapters into Wildfly or JBoss EAP, we would automatically include Keycloak client jars into EVERY application irregardless if you were using Keycloak or not.
-These libraries are now only added to your deployment if you have keycloak authentication turned on for that adapter (via the subsystem, or auth-method in web.xml 
+Previously, if you had installed our saml or oidc keycloak subsystem adapters into Wildfly or JBoss EAP, we would automatically include {{book.project.name}} client jars into EVERY application irregardless if you were using {{book.project.name}} or not.
+These libraries are now only added to your deployment if you have {{book.project.name}} authentication turned on for that adapter (via the subsystem, or auth-method in web.xml
```
Comment (stianst) [inline topics/MigrationFromOlderVersions.adoc]: Leave as 'keycloak authentication'. The authentication method is called 'keycloak' both in Keycloak and RH-SSO.

File: topics/MigrationFromOlderVersions.adoc [hunk 2]
```diff
@@ -104,15 +104,17 @@ You can turn off this behavior check adapter config switches.
...
 ==== Migrating to 1.8.0
 
 ===== Admin account
 
 In previous releases we shipped with a default admin user with a default password, this has now been removed.
 If you are doing a new installation of 1.8 you will have to create an admin user as a first step.
-This can be done easily by following the steps in <<_create_admin_user,Admin User>>. 
+This can be done easily by following the steps in 
+
+// DOCS REMARK: Please add the chapter as the referenced topic does not exist. <<_create_admin_user,Admin User>>.
```
Comment (stianst) [inline topics/MigrationFromOlderVersions.adoc]: Please don't include inline comments like this. Any issues should be reported in JIRA.

File: topics/MigrationFromOlderVersions.adoc [hunk 3]
```diff
@@ -332,14 +336,16 @@ Again you don't need to care about migration of database from previous version s
...
 Then you can simply paste this as allowed Redirect URI to the admin console of 3rd party provider (IE.
 Facebook admin console). 
 
 ==== Migrating from 1.1.0.Beta2 to 1.1.0.Final
 
 * WEB-INF/lib
-+`standalone/configuration/providers`<<_providers,+providers>>
++`standalone/configuration/providers` 
+
+// DOCS REMARK: Cross Reference not resolved. Please check and update <<_providers,+providers>>
```
Comment (stianst) [inline topics/MigrationFromOlderVersions.adoc]: Please don't include inline comments like this. Any issues should be reported in JIRA.
 Add a line note

File: topics/MigrationFromOlderVersions.adoc [hunk 4]
```diff
@@ -68,10 +68,10 @@ We've moved the themes and providers directories from `standalone/configuration/
 If you have added custom themes and providers you need to move them to the new location.
 You also need to update `keycloak-server.json` as it's changed due to this. 
 
-===== Adapter Subsystems only bring in dependencies if keycloak is on
+===== Adapter Subsystems only bring in dependencies if {{book.project.name}} is on
```
Comment (stianst) [inline topics/MigrationFromOlderVersions.adoc]: This needs rewriting, but should be left as 'keycloak'. The authentication method is called 'keycloak' both in Keycloak and RH-SSO.

File: topics/sso-protocols/oidc.adoc [hunk 5]
```diff
@@ -55,7 +55,9 @@ to provide a client secret when they exchange the temporary codes for tokens.  _
 _Public_ clients are perfectly fine so long as HTTPS is strictly enforced and you are very strict about what redirect URIs are registered for the
 client.  HTML5/Javascript clients actually always have to be _public_ clients because there is no way to transmit the client secret to them in a secure
 manner.  Again, this is ok so long as you use HTTPS and strictly enforce redirect URI registration.  This guide goes more detail
-into this in the <<fake/../../client.adoc,Client>> chapter.
+into this in the 
+
+// DOCS REMARK: Please update the cross-reference as it does not resolve correctly. <<fake/../../client.adoc,Client>> chapter.
```
Comment (stianst) [inline topics/sso-protocols/oidc.adoc]: Please remove comment. Changes/issues needs to be tracked in JIRA not as comments in docs

**Comment by aasingh** [comment]
I have made the necessary changes. 
1. Revert all the changes to the Migration chapter.
2. Enabled the conditional tag for the migration chapter in the SUMMARY.adoc

**Pull request merged**